Woodoku/src/Entity/WoodokuBoard.py

```diff
@@ -25,6 +25,9 @@ class _WoodokuBoardRepresentation:
     def __init__(self):
         self.__board = np.full((N, N), False)
 
+    def board(self):
+        return self.__board
+
```

Comment:
Exposing a private variable is never considered as a good approach. Since this is a data class that hides the underlining implementation, `__board` should not be exposed in any case.